lib/strategy/shex_context.json

```diff
@@ -0,0 +1,1171 @@
+{
```

Comment:
Maybe it should be moved somewhere else but I don't know. As a string in the strategy file it take to much space IMO.